src/test/prob/normal/normal_sufficient_test.hpp

```diff
@@ -0,0 +1,117 @@
+// Arguments: Doubles, Doubles, Ints, Doubles, Doubles

```

Comment:
I don't know: poisson_log and poisson_log_log tests are in the same folder, binomial_log and binomial_logit_log too, but neg_binomial_log and neg_binomial_2_log aren't.